components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -168,6 +172,17 @@ void BravePrefProvider::MigrateShieldsSettings(bool incognito) {
   MigrateShieldsSettingsV2ToV3();
 }
 
+void BravePrefProvider::EnsureNoWildcardEntries() {
```

Comment:
It seems you also need to add one 'if' in SetWebsiteSettingInternal. You can check it by clicking on 'Reset Settings'

Reply:
one `if` for what? Not sure what Reset Settings has to do with this?

Reply:
everything seems to be working correctly, I'm not understanding what the issue is here

Reply:
I was wrong.  When reset, it will write CONTENT_SETTING_DEFAULT to (*, *). It is what we want.